src/index/components/TabPanel.vue

```diff
@@ -156,12 +162,24 @@
             >
             <div class="coin-name-container">
               <span class="coin-symbol">{{ coin.symbol }}</span>
-              <span class="coin-name">{{ coin.name }}</span>
+              <span class="coin-name">
+                <span
+                  v-if="!tokenPrices[coin.symbol]"
+                  class="skeleton-loader price-skeleton"
+                />
+                <span v-else>${{ formatPrice(tokenPrices[coin.symbol]) }}</span>
+              </span>
             </div>
           </div>
           <div class="coin-balance-value">
             <span class="coin-balance">{{ coin.balance }}</span>
-            <span class="coin-value">${{ calculateUsdValue(coin) }} USD</span>
+            <span class="coin-value">
+              <span
+                v-if="!tokenPrices[coin.symbol]"
```

Comment:
if the token is not listed in KoinDX do not show the loader. In this cases we can leave it empty.